### PR TITLE
Edited plusminus template for issue 105

### PR DESCRIPTION
**Pull request by coverney**
I was wondering whether the comment, "This analysis supports two variables: a +/- variable and a tuning variable" is still applicable.

### CODE REVIEW

File: template_analysis/plusminus_template.m [hunk 1]
```diff
@@ -28,18 +28,19 @@
 % If your distribution is more complex or less complex, you can change the number of components
 % AP=setNumGaussianComponents(AP,3);
 
-% Make a map of the batches of plus/minus comparisons to test
+% Make a map of the batches comparisons to test, add in a list of batch
+% names (i.e. {'+', '-'}) to signify possible sets.
```
Comment (jakebeal) [inline template_analysis/plusminus_template.m]: "e.g.," not "i.e."

File: template_analysis/plusminus_template.m [hunk 2]
```diff
@@ -28,18 +28,19 @@
...
 % This analysis supports two variables: a +/- variable and a "tuning" variable
 stem1011 = '../example_assay/LacI-CAGop_';
 batch_description = {...
- {'Lows';'BaseDox';
+ {'Lows';'BaseDox';{'+', '-'};
```
Comment (jakebeal) [inline template_analysis/plusminus_template.m]: Don't forget to update the template with your 3-condition example.

**Review by jakebeal — CHANGES_REQUESTED** [review]

File: template_analysis/plusminus_template.m (hunk 2 above)
Comment (coverney) [reply template_analysis/plusminus_template.m]: So I should include a second batch_description with 3 conditions in the same template?

**Review by jakebeal — APPROVED** [review]

**Comment by jakebeal** [comment]
Actually, I ended up just doing this myself in order to help myself review your other pull request.

File: template_analysis/plusminus_template.m (hunk 2 above)
Comment (jakebeal) [reply template_analysis/plusminus_template.m]: I went and did it myself in order to help review the other pull request. :-)

**Pull request merged**